python/vosk/transcriber/cli.py

```diff
@@ -12,6 +12,9 @@
 parser.add_argument(
         '--model', '-m', type=str,
         help='model path')
+parser.add_argument(
+        '--vosk-server', '-vs', default=False, type=bool,
```

Comment:
Just --server. Also make it a string (URL) of the server with default value localhost:2700